src/frontend/src/components/TimeSlot.tsx

```diff
@@ -22,14 +24,16 @@ const TimeSlot: React.FC<TimeSlotProps> = ({
   onMouseDown,
   onMouseEnter,
   onMouseUp,
-  onMouseOver
+  onMouseOver,
+  onClick,
+  selected = false
 }) => {
   return (
     <Box
       sx={{
         height: small ? '25px' : '4.7vh',
         width: small ? '81px' : '12.2%',
-        backgroundColor,
+        backgroundColor: selected ? '#9b9b9b' : backgroundColor,
```

Comment:
instead of changing the backgroundColor to this new color, can we change the border color to a yellowish color like a highlight